src/plugins/credential_management/public/components/credential_table/credentials_table.tsx

```diff
@@ -112,8 +114,46 @@ export const CredentialsTable = ({ canSave, history }: Props) => {
       dataType: 'string' as const,
       sortable: ({ sort }: { sort: string }) => sort,
     },
+    {
+      field: 'credentialType',
+      name: 'Credential Type',
+      truncateText: true,
+      mobileOptions: {
+        show: false,
+      },
+    },
   ];
 
+  const onSelectionChange = (selectedCredentials: CredentialsTableItem[]) => {
+    setSelectedCredentials(selectedCredentials);
+  };
+
+  const selection = {
+    onSelectionChange,
+  };
+
+  const renderDeleteButton = () => {
+    if (selectedCredentials.length === 0) {
+      return;
+    }
+
+    return (
+      <EuiButton color="danger" iconType="trash" onClick={onClickDelete}>
+        Delete {selectedCredentials.length} Credentials
+      </EuiButton>
+    );
+  };
+
+  const onClickDelete = () => {
+    Promise.resolve(deleteCredentials(savedObjects.client, selectedCredentials)).then(function () {
```

Comment:
TODO: add validation logic
- prevent user to delete credential has been used in any datasource.

Reply:
https://github.com/opensearch-project/OpenSearch-Dashboards/issues/1876